test/functional/mining_prioritisetransaction.py

```diff
@@ -94,12 +96,12 @@ def test_diamond(self):
 
         self.log.info("Test priority while txs are in mempool")
         raw_before = self.nodes[0].getrawmempool(verbose=True)
-        fee_delta_b = Decimal(9999) / COIN
-        fee_delta_c_1 = Decimal(-1234) / COIN
-        fee_delta_c_2 = Decimal(8888) / COIN
-        self.nodes[0].prioritisetransaction(txid=txid_b, fee_delta=int(fee_delta_b * COIN))
-        self.nodes[0].prioritisetransaction(txid=txid_c, fee_delta=int(fee_delta_c_1 * COIN))
-        self.nodes[0].prioritisetransaction(txid=txid_c, fee_delta=int(fee_delta_c_2 * COIN))
+        fee_delta_b = satToBtc(Decimal(9999))
+        fee_delta_c_1 = satToBtc(Decimal(-1234))
+        fee_delta_c_2 = satToBtc(Decimal(8888))
```

Comment:
Doesn't `satToBtc` accept an `int`?

Reply:
Yes, passing sats as a decimal should never be necessary.